site/docs/community.md

```diff
@@ -73,3 +73,12 @@ Iceberg has four mailing lists:
     - [Archive](https://lists.apache.org/list.html?issues@iceberg.apache.org)
 * **Private**: <private@iceberg.apache.org> -- private list for the PMC to discuss sensitive issues related to the health of the project
     - [Archive](https://lists.apache.org/list.html?private@iceberg.apache.org)
+
+
+## Setting up IDE and Code Style
+
+### Configuring Code Formatter for IntelliJ IDEA
+
+Navigate to **File > Settings > Editor > Code Style > Java**. Click on the gear wheel and select **Import Scheme** to import IntelliJ IDEA XML code style settings.
+Point to [intellij-java-palantir-style.xml](../../.baseline/idea/intellij-java-palantir-style.xml) and hit **OK**. See also the [IntelliJ docs](https://www.jetbrains.com/help/idea/copying-code-style-settings.html)
+for additional details.
```

Comment:
Can we at least mentioned that user needs to go to "Code > Reformat Code/File" to trigger it? Besides, I love the way that it formats the code automatically every time you save a file. I used it a lot in my Scala coding, but looks like we need a Macros to support it in IntelliJ, which is kind of cumbersome to be here.

Reply:
done